apps/govquests-frontend/package.json

```diff
@@ -10,6 +10,7 @@
     "check-types": "tsc --noEmit"
   },
   "dependencies": {
+    "@radix-ui/react-icons": "^1.3.0",
```

Comment:
I would take `lucide-react` https://lucide.dev/icons/

Reply:
@devjoaov Got it, but why so? Just curious to know why you prefer this lib?